vigiliae-christianae.csl

```diff
@@ -0,0 +1,776 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only">
```

Comment:
@carljohanberglund, it looks like this journal publishes in English, German, and French, so we probably don't want to fix the style to a single language, right?